src/transaction/log_manager.c

```diff
@@ -4710,6 +4729,10 @@ log_change_tran_as_completed (THREAD_ENTRY * thread_p, LOG_TDES * tdes, LOG_RECT
 static void
 log_append_commit_log (THREAD_ENTRY * thread_p, LOG_TDES * tdes, LOG_LSA * commit_lsa)
 {
+  if (prm_get_bool_value (PRM_ID_SUPPLEMENTAL_LOG) == true)
+    {
+      log_append_supplement_user (thread_p, tdes, LOG_PRIOR_LSA_WITHOUT_LOCK);
```

Comment:
It seems to be put into `log_append_donetime_internal()` with this in `log_append_abort_log()`.

Reply:
OK, Thanks. It could be replaced.